views/templates/block/checklist.twig

```diff
@@ -133,30 +129,84 @@
                         {% endif %}
                     </td>
                 </tr>
+                {% if not checkPhpVersionCompatibility %}
+                  <tr>
+                    <td>{{ 'Your current PHP version isn\'t compatible with your PrestaShop version.'|trans }}</td>
+                    <td>{{ icons.nok }}</td>
+                  </tr>
+                {% endif %}
+                {% if not checkApacheModRewrite %}
+                  <tr>
+                    <td>{{ 'Apache mod_rewrite is disabled.'|trans }}</td>
+                    <td>{{ icons.nok }}</td>
+                  </tr>
+                {% endif %}
+                {% if notLoadedPhpExtensions|length > 0 %}
+                  <tr>
+                    <td>
+                      {{ 'The PHP extension are not installed: %s.'|trans([notLoadedPhpExtensions|join(', ')]) }}
+                    </td>
+                    <td>{{ icons.nok }}</td>
+                  </tr>
+                {% endif %}
+                {% if not checkMemoryLimit %}
+                <tr>
+                  <td>{{ 'PHP memory_limit is inferior to 256 MB.'|trans }}</td>
+                  <td>{{ icons.nok }}</td>
+                </tr>
+                {% endif %}
+                {% if not checkFileUploads %}
                 <tr>
+                  <td>{{ 'PHP file_uploads configuration is disabled.'|trans }}</td>
+                  <td>{{ icons.nok }}</td>
+                </tr>
+                {% endif %}
+                {% if notExistsPhpFunctions|length > 0 %}
+                  <tr>
                     <td>
-                        {% if isPrestaShopReady %}
-                            {{ 'PrestaShop requirements are satisfied.'|trans }}
-                        {% else %}
-                            {{ 'PrestaShop requirements are not satisfied. [1]See details[/1] or [2]ignore[/2].'|trans({
-                                '[1]': '<a href="https://devdocs.prestashop.com/1.7/basics/installation/system-requirements/">',
-                                '[/1]': '</a>',
-                                '[2]': '<a href="' ~ currentIndex ~ '&token=' ~ token ~ '&ignorePsRequirements=1">',
-                                '[/2]': '</a>',
-                            })|raw }}
-                        {% endif %}
+                      {{ 'PHP functions are not installed: %s.'|trans([notExistsPhpFunctions|join(', ')]) }}
```

Comment:
Same here:

The PHP function fopen, fclose, fread, fwrite, rename, file_exists, unlink, rmdir, mkdir, getcwd, chdir, chmod, imagecreatetruecolor is/are not installed. (display only the one(s) that are not installed)

So:

'The following PHP functions are not installed: %s.' and 'The following PHP function is not installed: %s.'

Reply:
Fixed